java/server/src/org/openqa/grid/web/servlet/HubStatusServlet.java

```diff
@@ -136,16 +136,14 @@ private JsonObject getResponse(HttpServletRequest request) throws IOException {
   private JsonObject getSlotCounts() {
     int freeSlots = 0;
     int totalSlots = 0;
+    int usedSlots = 0;
 
     for (RemoteProxy proxy : getRegistry().getAllProxies()) {
-      for (TestSlot slot : proxy.getTestSlots()) {
-        if (slot.getSession() == null) {
-          freeSlots += 1;
-        }
-
-        totalSlots += 1;
-      }
+      totalSlots += proxy.getTestSlots().size() > proxy.getMaxNumberOfConcurrentTestSessions() ?
```

Comment:
Don't we still want += here? It is summing all the slots/sessions of all the nodes connected to the hub.